docs/src/user/build.md

```diff
@@ -34,23 +34,23 @@ Only `torch` is needed. You can skip the installation of `torchvision` and `torc
 ### Build and install libriichi
 > Working directory: `$MORTAL_ROOT`
 ```shell
-$ cargo build -p libriichi --lib --release
+cargo build -p libriichi --lib --release
 ```
 
 For Linux
 ```shell
-$ cp target/release/libriichi.so mortal/libriichi.so
+cp target/release/libriichi.so mortal/libriichi.so
 ```
 
 For Windows (MSYS2)
 ```shell
-$ cp target/release/riichi.dll mortal/libriichi.pyd
+cp target/release/riichi.dll mortal/libriichi.pyd
 ```
 
 ### Test the environment
 > Working directory: `$MORTAL_ROOT/mortal`
 ```shell
-$ python
```

Comment:
The dollar sign (command line prompt) is intentional so that commands and output can be separated.

Reply:
For eyes, yes the dollar sign makes the command more visually recognizable.

In the browser, when hovering over the code box, there comes a button that says "copy to clipboard".
<img width="881" alt="image" src="https://github.com/Equim-chan/Mortal/assets/7020805/0e653b73-8641-414d-aeac-2c35148a979c">
But if you do copy and paste into a terminal, you'll be executing a command starting with `$`, which the shell cannot recognize.

Maybe worth disabling the "copy to clipboard" button somewhere?